Packs/InfoArmor_VigilanteATI/Integrations/InfoArmorVigilanteATI/InfoArmorVigilanteATI.py

```diff
@@ -0,0 +1,744 @@
+    """
+    filter only the params that have values
+    """
+    return dict((k, v) for k, v in params_dict.items() if v is not None)
+
+
+def query_ecrime_intelligence_database(query, q_forum, q_start_data, limit, re_token=None):
+    params = remove_none_params({
+        'query': query,
+        'q_forum': q_forum,
+        'q_start_data': q_start_data,
+        'limit': limit,
+        're_token': re_token
+    })
+
+    res = http_request('get', 'api/1/ecrime/posts/query', 'ecrime.posts.query', params)
+    return res.json()
+
+
+def query_ecrime_intelligence_database_command(args):
+    query = args.get('query')
+    q_forum = args.get('q_forum')
+    q_start_data = args.get('q_start_data')
+    limit = int(args.get('limit', 10))
+    re_token = args.get('re_token')
+
+    results = query_ecrime_intelligence_database(query, q_forum, q_start_data, limit, re_token)
+    posts = results.get('posts')
+    for post in posts:
+        post['title'] = base64.b64decode(post['title']).decode('utf8')
+        post['post'] = base64.b64decode(post['post']).decode('utf8')
+
+    markdown = tableToMarkdown('ECrime Posts', posts)
+    markdown += '\n**Total Count**: {}'.format(results.get('count'))
+    markdown += '\n**Next Page Token**: \n{}'.format(results.get('re_token'))
+
+    outputs = {
+        'VigilanteATI.ECrimePosts': posts
+    }
+    if results.get('re_token'):
+        outputs['VigilanteATI.ECrimeQueryToken(true==true)'] = results.get('re_token')
+
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': results,
+        'HumanReadable': markdown,
+        'EntryContext': outputs
+    })
+
+
+def list_account_credentials(leak_id, limit, domains, token=None):
+    params = remove_none_params({
+        'leak_id': leak_id,
+        'limit': limit,
+        'domains': domains,
+        'token': token
+    })
+
+    res = http_request('get', 'api/1/leaks/get', 'leaks.get', params)
+    return res.json()
+
+
+def query_accounts(account_identifier, limit, days_ago):
+    if days_ago is not None:
+        days_ago = int(days_ago)
+
+    body = remove_none_params({
+        'account_identifier': account_identifier,
+        'limit': limit,
+        'daysAgo': days_ago
+    })
+
+    res = http_request('post', 'api/3/accounts/query', 'accounts.query', None, data=body)
+    return res.json()
+
+
+def query_accounts_command(args):
+    emails = argToList(args.get('emails'))
+    days_ago = args.get('days_ago')
+    limit = args.get('limit')
+
+    results = query_accounts(emails, limit, days_ago)
+    accounts = results.get('results')
+    for i, _ in enumerate(accounts):
+        account = accounts[i]
+        account['email'] = account['plain']
+        del account['plain']
+
+    markdown = tableToMarkdown('Leaks related to email accounts \n{}'.format('\n'.join(emails)),
+                               accounts, ['leak_id', 'email', 'password', 'source_type', 'type_id'])
```

Comment:
We print the `password` in lear text?

Reply:
The accounts are leaked accounts, thus the information is exposed.